x-pack/plugin/sql/qa/server/src/main/resources/agg.csv-spec

```diff
@@ -8,17 +8,17 @@ singlePercentileWithoutComma
 SELECT gender, PERCENTILE(emp_no, 90 + 7) p1 FROM test_emp GROUP BY gender;
 
 gender:s             | p1:d
-null                 |10019.0           
-F                    |10099.51          
+null                 |10019.0
+F                    |10099.51
```

Comment:
there's lots of WS changes that snuck in, i'll remove them with 1st review update.

Reply:
Removed.